### PR TITLE
Get txType for requested tx

### PR DESCRIPTION
**Pull request by PJLewandowski**

### CODE REVIEW

File: src/script/standard.h [hunk 1]
```diff
@@ -86,6 +86,7 @@ enum vaulttxnstatus
     TX_PENDING,
     TX_CONFIRMED,
     TX_RECOVERED,
+	TX_UNKNOWN
```
Comment (harzo) [inline src/script/standard.h]: indentation

File: src/rpc/rawtransaction.cpp [hunk 2]
```diff
@@ -644,7 +648,10 @@ static UniValue decoderawtransaction(const JSONRPCRequest& request)
     }
 
     UniValue result(UniValue::VOBJ);
-    TxToUniv(CTransaction(std::move(mtx)), uint256(), result, false);
+    auto tx = CTransaction(std::move(mtx));
+    vaulttxnstatus txStatus = TX_UNKNOWN;
+    vaulttxntype txType = GetVaultTxTypeNonContextual(tx);
+    TxToJSON(tx, uint256(), txType, txStatus, result);
```
Comment (harzo) [inline src/rpc/rawtransaction.cpp:654]: do `TxToJSON` and `TxToUniv` both return JSON?

**Review by harzo — CHANGES_REQUESTED** [review]

File: src/script/standard.h (hunk 1 above)
Comment (PJLewandowski) [reply src/script/standard.h]: Done

File: src/rpc/rawtransaction.cpp (hunk 2 above)
Comment (PJLewandowski) [reply src/rpc/rawtransaction.cpp:654]: Yup, TxToJSON even call TxToUniv internally at the beginning.

**Review by harzo — APPROVED** [review]

**Pull request merged**